plugin/README.md

```diff
@@ -14,4 +14,5 @@ Each of these plugins has a corresponding example in [examples](../examples/hook
 ├── <a href="./kgmetrics">kgmetrics</a> — plug-in go-metrics to use with `kgo.WithHooks`
 ├── <a href="./kprom">kprom</a> — plug-in prometheus metrics to use with `kgo.WithHooks`
 └── <a href="./kzap">kzap</a> — plug-in uber-go/zap to use with `kgo.WithLogger`
```

Comment:
Mind fixing the first character here to use one of the sidways T things?

Reply:
Voilà.